Azure/Azure-AppServices-Diagnostics-Portal
# Update gist version separate component

## Overview


New functionality to show list of detector referencing a particular gist. When gist is updated, it gives user the ability to update detectors referencing that gist.
Provides efficient solution for updating detectors referencing a particular gist, and reduces time taken for user to do so

## Related Work Item

User Story 2266 : Enable bulk update of detectors to point to latest version of GIST
https://dev.azure.com/app-service-diagnostics-portal/app-service-diagnostics-portal/_workitems/edit/2266

## Type of change




## Solution description

### This PR:
Changes to code in onboarding-flow.component.html + onboarding-flow.component.ts: created new "update detector references" button and dialog)
Created new component update-detector-references: .html + .ts files changed. All the logic for displaying detector references and updating is in this file. 
Changes to code in fab-data-table.component.html + fab-data-table.component.ts: returns selected items from the data table through use of a new button which can be taken as input 





## How Can This Be Tested? <span>&#128269;</span>



Navigate to a gist that has the code loaded in local environment, and press "update detector references" button to see list of detector references. 
Once updates to gist have been made, refresh the page to see changes in table and update detectors. 
Once detectors have been updated through new functionality and pr has been merged, refresh the page to see changes of status. 
(remember to api/deploy changes and then refresh)

## Checklist <span>&#9989;</span>

## Screenshots Before Fix (if appropriate):

## Screenshots After Fix (if appropriate):

## Extra Notes
